### PR TITLE
Add the Sendgrid api config 

### PR DESCRIPTION
**Pull request by raviitsoft**
add the sendgrid API config

### CODE REVIEW

File: {{cookiecutter.project_slug}}/Pipfile [hunk 1]
```diff
@@ -19,6 +19,7 @@ django-bootstrap4 = "~=0.0.8"
 django-allauth = "~=0.39.1"
 django-rest-auth = "~=0.9.5"
 django-extensions = "~=2.1.9"
+django-sendgrid-v5 = "*"
```
Comment (driverdan) [inline {{cookiecutter.project_slug}}/Pipfile]: What does this library add that sending through SMTP doesn't give us?

File: {{cookiecutter.project_slug}}/{{cookiecutter.project_slug}}/settings.py [hunk 2]
```diff
@@ -190,12 +190,23 @@
...
 AUTH_USER_MODEL = "users.User"
 
-EMAIL_HOST = env.str("EMAIL_HOST", "smtp.sendgrid.net")
-EMAIL_HOST_USER = env.str("SENDGRID_USERNAME", "")
-EMAIL_HOST_PASSWORD = env.str("SENDGRID_PASSWORD", "")
+
+# Configuration sendgrid api start
+EMAIL_BACKEND = 'sendgrid_backend.SendgridBackend'
+
+DEFAULT_FROM_EMAIL = 'Crowdbotics <noreply@crowdbotics.com>'
```
Comment (driverdan) [inline {{cookiecutter.project_slug}}/{{cookiecutter.project_slug}}/settings.py]: This should be `support@crowdbotics.com`. Nonexistent email addresses are a bad practice.

Comment (raviitsoft) [reply {{cookiecutter.project_slug}}/{{cookiecutter.project_slug}}/settings.py]: I am just using demo purpose you can change according to yourself, whatever you need.

File: {{cookiecutter.project_slug}}/Pipfile (hunk 1 above)
Comment (raviitsoft) [reply {{cookiecutter.project_slug}}/Pipfile]: I think maybe not support properly so I was at this package and everything works fine

**Pull request closed without merging**